group-actors/src/main/java/org/sunbird/service/GroupService.java

```diff
@@ -14,9 +14,10 @@
 
   Map<String, Object> readGroup(String groupId) throws BaseException;
 
-  GroupResponse readGroupWithActivities(String groupId) throws Exception;
+  GroupResponse readGroupWithActivities(String groupId, Map<String, Object> reqContext)
+      throws Exception;
 
-  void readGroupActivities(Map<String, Object> dbResGroup);
+  void readGroupActivities(Map<String, Object> dbResGroup, Map<String, Object> reqContext);
 
   List<GroupResponse> searchGroup(Map<String, Object> searchFilter) throws BaseException;
```

Comment:
Context need to be passed

Reply:
We use MDC to log traceId in groups, so there is no requirement to pass request context in each method to log traceId